ui/base-tool/src/main/java/org/cru/godtools/base/tool/activity/BaseSingleToolActivity.kt

```diff
@@ -86,32 +81,23 @@ abstract class BaseSingleToolActivity(
         downloadManager.cacheTranslation(toolCode, locale)
     }
 
-    override fun determineActiveToolState() = when {
-        !hasTool() -> STATE_LOADED
```

Comment:
Is this case still being accounted for somewhere?

Reply:
I don't think this case was originally necessary. the only time `hasTool` would return false would be in `AemArticleActivity`, but that class had it's own implementation of `determineActiveToolState()`.

This logic was probably written before it got overridden in `AemArticleActivity`